extensions/wikia/EmbeddableDiscussions/scripts/sharing.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Functionality for social network sharing for Embeddable Discussions extension
+ */
+define('EmbeddableDiscussionsSharing',
+	['jquery'],
+	function ($) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)